### PR TITLE
Support inline artifacts.

### PR DESCRIPTION
**Pull request by shrinandj**
Artifacts can now be inline with the sensor yaml. See ./examples/inline-sensor.yaml
as an example.

Refs #41

### CODE REVIEW

File: pkg/apis/sensor/v1alpha1/types.go [hunk 1]
```diff
@@ -382,7 +382,8 @@ type URI struct {
 
 // ArtifactLocation describes the location for an external artifact
 type ArtifactLocation struct {
-	S3 *S3Artifact `json:"s3,omitempty" protobuf:"bytes,1,opt,name=s3"`
+	S3     *S3Artifact `json:"s3,omitempty" protobuf:"bytes,1,opt,name=s3"`
+	Inline string      `json:"inline,omitempty" protobuf:"bytes,3,opt,name=inline"`
```
Comment (magaldima) [inline pkg/apis/sensor/v1alpha1/types.go]: i think the byte location should be 2

Comment (shrinandj) [reply pkg/apis/sensor/v1alpha1/types.go]: Done.

**Comment by magaldima** [comment]
can you rebase this on top of master?

**Comment by shrinandj** [comment]
I am trying to but running into some merge conflicts ... will update the PR.

**Pull request merged**